src/utils.rs

```diff
@@ -96,6 +96,11 @@ pub fn write_file<P: AsRef<Path>, C: AsRef<[u8]>>(path: P, contents: C) -> Resul
     Ok(())
 }
 
+pub fn create_dir_all<P: AsRef<Path>>(path: P) -> Result<()> {
```

Comment:
I thought there might be no need to bother wrapping it in a function, is there any reason for this?

Reply:
In the case of an error I want that the error message contains the path that should have been created, In the past it had been difficult to find out where the error is happening without this information. I could add with_context to every call, but having it in one place made more sense to me.